pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -123,6 +123,17 @@ def test_loc_multiindex_missing_label_raises(self):
         with pytest.raises(KeyError, match=r"^2$"):
             df.loc[2]
 
+        # Lists with missing labels do not raise:
```

Comment:
these should probably be separate tests, although it does mean duplicating dataframe setup.

Reply:
I think that's a manageable test... and not worth a fixture (if that's what you mean)

Reply:
No, sorry, I misread "it does mean" in "it doesn't mean". Not a fan of duplicating the setup to be honest.